Gruntfile.js

```diff
@@ -1,70 +1,64 @@
 module.exports = function(grunt) {
 
-	grunt.initConfig({
-		pkg: grunt.file.readJSON('package.json'),
-		jshint: {
```

Comment:
I'm not sure JSHint has been made redundant by JSCS. I re-enabled it and it highlighted some issues. What do you think?